src/librustc_session/options.rs

```diff
@@ -1049,6 +1049,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "validate MIR after each transformation"),
     verbose: bool = (false, parse_bool, [UNTRACKED],
         "in general, enable more debug printouts (default: no)"),
+    span_debug: bool = (false, parse_bool, [UNTRACKED],
+        "Forwards proc_macro::Span's `Debug` impl to `Span`"),
```

Comment:
See the comments above or below this list :)

Reply:
The comments say to keep options in alphabetic order and to add tests to `src/librustc_interface/tests.rs`.